CAIPs/caip-3.md

```diff
@@ -0,0 +1,69 @@
+---
+caip: 3
+title: The Ethereum interface for CAIP-2
```

Comment:
Blockchain reference for Ethereum interface

Reply:
Should we rally drop the link to CAIP-2?

Reply:
Yes, because the title shouldn't describe the requirements. We just refer it on `require: 2`. Additionally standards are modular because they can be used individually. I could just make use of CAIP-3 for something other chan CAIP-2 for example. I think @ligi would be able to provide more clarity on this than me

Reply:
Implemented in https://github.com/ChainAgnostic/CAIPs/pull/9/commits/c803d602f33596b7660fba63b91c5dd7bebea8d5. Please bear in mind that the goal is to merge the two Cosmos documents into one document one way or another, so "Blockchain Reference for Cosmos (Hashed Variant)" should be considered a temporary working title. Resolving this.